### ISSUE TITLE
Editor Performance Problem After Enable Side By Side Mode

### ISSUE THREAD

**Issue by 7upcat** [sparksuite/simplemde-markdown-editor#206]
After enable  'toggleSideBySide' mode  will  registry 'update' callback function(which should be removed if the side by side mode off ).
When  the markdown file is big the editor will be very very slow  ....

```
cm.on("update", function() {
    preview.innerHTML = editor.options.previewRender(editor.value(), preview);
```

**Comment by WesCossick** [comment]
Are you saying that the problem is that the editor is slow when the Markdown content is large and side-by-side mode is active, or that the problem is that the editor doesn't destroy the update listener? Or both?

**Comment by 7upcat** [comment]
I think realtime update rending is the reason cause slow.
Mybe markdown doc is not that large, but Iam try to support asciidoc I
found the performance problem.

**Comment by aruizca** [comment]
I think this might be related to my issue #194

**Comment by 7upcat** [comment]
Javascript is single-threaded .If rending on client need take time(seconds)  and rending operation is so much frequency. The user will wait for few seconds for every typed......

Maybe can add a option specify render interval : ) e.g:

``` javascript
if(currentTimestamp-lastrRendTimestamp<operation.rendeInterval){
    //do not rend 
}else{
  //do rend job
  lastrRendTimestamp =new Date().getTimestamp();
}
```

Maybe can using setTimout() make the rending and typing asynchronize.

**Comment by WesCossick** [comment]
I think it has to do with something you tried when supporting asciidoc. When I use this [very large Markdown text](http://daringfireball.net/projects/markdown/syntax.text) in SimpleMDE's side-by-side mode on the [demo website](http://nextstepwebs.github.io/simplemde-markdown-editor), there is no performance issues.

**Comment by WesCossick** [comment]
Fixed in PR #211
